### PR TITLE
Promote/DO App Platform Example

### PR DESCRIPTION
**Pull request by black-bryan**
Promote the scim-example for DigitalOcean App Platform from the Beta examples directory.

- This process of deployment has been used by many customers and since this is a quick and easy low cost SCIM bridge deployment, it is time to move these deployment options out of Beta.
  - Link to DigitalOcean App Platform 1Password SCIM Bridge deployment using the DO portal
  - Promoted DigitalOcean App Platform using the DO command line tool and the 1Password CLI from the beta directory

Other changes:
- Introducing concept on main README.md and PREPARATION.md of Container-as-a-Service deployment, differentiating from Advanced and One-Click (Marketplace) deployments.
  - Detailing on the  PREPARATION.md, how DNS and certificate management are handled by the CaaS infrastructure automatically.

### CODE REVIEW

**Comment by black-bryan** [comment]
As part of this promotion of the DO App Platform out of the beta directory, we also need to ensure that the path for the .yaml is updated on this support guide for obtaining the YAML, [1Password SCIM Bridge deployment examples repo on GitHub](https://support.1password.com/cs/scim-deploy-digitalocean-ap/#step-2-create-resources-in-digitalocean)

**Review by accraw — APPROVED** [review]

File: do-app-platform-op-cli/README.md
Comment (scottisloud) [inline do-app-platform-op-cli/README.md:5]:
```suggestion
Alternatively, you can deploy the [1Password SCIM Bridge to DigitalOcean's App Platform Container-as-a-Service environment using the DigitalOcean portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/) instead of the command line. 
```
Correction by adding comma after "alternatively". The rest is a suggestion, take or leave!

File: do-app-platform-op-cli/README.md
Comment (scottisloud) [inline do-app-platform-op-cli/README.md:1]: (oddly I can't comment on lines not changed in the PR)

Any URLs in commands throughout this document will need to be updated to the new path of the deployment manifest. Many still point to the beta subdirectory.

**Review by scottisloud — CHANGES_REQUESTED** [review]
Some minor suggestions and a few corrections/updated (e.g., urls pointing to a beta subdirectory).

Beyond that, this looks great.

File: do-app-platform-op-cli/README.md
Comment (black-bryan) [reply do-app-platform-op-cli/README.md:1]: That is a great callout. This time I will be sure to follow through and test the URLs after merging as well.

File: README.md
Comment (ag-adampike) [inline README.md:5]: I would prefer that we standardize on "marketplace apps" for these deployment options:

```suggestion
## Marketplace apps
```

**Review by ag-adampike — CHANGES_REQUESTED** [review]
Looking good overall with a few requested changes and suggestions.

File: README.md
Comment (black-bryan) [reply README.md:5]: I love this suggestion!

File: do-app-platform-op-cli/README.md
Comment (black-bryan) [reply do-app-platform-op-cli/README.md:1]: I had the same issue, I had to commit the changes from file. Updated many beta paths

File: do-app-platform-op-cli/google-workspace/workspace-settings.json
Comment (grellyd) [inline do-app-platform-op-cli/google-workspace/workspace-settings.json:1]: 🤔 Should this be `workspace-settings.json.example`?

**Review by grellyd — DISMISSED** [review]
Excellent! The simplicity of this MR speaks to the effectiveness of the Beta process.

**Comment by grellyd** [comment]
You may want to consider resolving https://github.com/1Password/scim-examples/pull/263 before merging this PR.

File: PREPARATION.md
Comment (Penderworth) [inline PREPARATION.md:27]: I think we can remove the hyphenation for CaaS and the capitalization here and in other instances. Also, it should be plural without "platforms". Usage elsewhere varies, but [Atlassian](https://www.atlassian.com/microservices/cloud-computing/containers-as-a-service) doesn't use hyphens and neither does [IBM](https://www.ibm.com/topics/containers-as-a-service) so I'd rather we stick to that. Right now it reads a little strange. I've also added "and" instead of a comma, and a contraction instead of "do not".

```suggestion
>Containers as a service, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to create a DNS record for these deployments.
```

File: PREPARATION.md
Comment (Penderworth) [inline PREPARATION.md:36]: To match the changes above, but for this context I've removed the examples since they were just mentioned a few paragraphs before:

```suggestion
>Containers as a service, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to use the optional certificate manager component.
```

File: PREPARATION.md
Comment (Penderworth) [inline PREPARATION.md:75]: To be a bit more direct with the recommendation, we can use an active voice:

```suggestion
For most custom deployments, we recommend that you clone this repository before you start the deployment. This ensures you have all the files you need and provides an opportunity for you to familiarize yourself with the contents of the deployment method you've selected.
```

File: README.md
Comment (Penderworth) [inline README.md:7]: As I read this, I wondered what "guided" means, since we have guides (tutorials) for both methods. We should imply that it's easier, which is more accurate:

```suggestion
If you'd prefer to deploy 1Password SCIM Bridge without the command line, you can use the Google Cloud Platform or DigitalOcean marketplace app:
```

I'm open to other suggestions here of course, but I do think that some readers may feel in over their head with all the commands.

File: README.md
Comment (Penderworth) [inline README.md:14]: To simplify this a bit for improved readability:

```suggestion
To customize your deployment, you can use containers as a service or create your own advanced deployment with the examples below as your base.
```

We explain CaaS above so we don't need to again here, similar to how we're not defining what "advanced deployment" means.

File: README.md
Comment (Penderworth) [inline README.md:22]: To match the suggestions above and make this a tad more concise. I also changed "few" to "minimal" since that clicks in my mind as "ooo this is going to be nice!"

```suggestion
### Containers as a service deployment options

Containers as a service (CaaS) can simplify your deployment by using the built-in tools of the CaaS for DNS and certificate management. This gives you an easy, low-cost SCIM bridge with minimal infrastructure management requirements.
```

File: README.md
Comment (Penderworth) [inline README.md:29]: To use an active voice here:

```suggestion
If you have particular requirements for your environment, we recommend an advanced deployment. These example configurations will give you a base to create the deployment from, as well as explain what 1Password SCIM Bridge needs to function and how to maintain your bridge once you've deployed it.
```

File: do-app-platform-op-cli/README.md
Comment (Penderworth) [inline do-app-platform-op-cli/README.md:1]:
```suggestion
# Deploy 1Password SCIM Bridge on DigitalOcean App Platform with 1Password CLI
```

File: do-app-platform-op-cli/README.md
Comment (Penderworth) [inline do-app-platform-op-cli/README.md:5]: "instead" of is implied by "alternatively". I also cleaned up the linked text a bit, and I think in this case it's okay for "container" to not be plural since it doesn't read well. I'm tempted to remove it from this link, though, since it's not explained in this article and folks may not have context to know what it means. So if you agree, there's a suggestion without it below:

```suggestion
Alternatively, you can [deploy 1Password SCIM Bridge to DigitalOcean's App Platform container as a service environment using the DigitalOcean portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/).
```

without CaaS:

```suggestion
Alternatively, you can [deploy 1Password SCIM Bridge to DigitalOcean App Platform using the DigitalOcean portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/).
```

**Review by Penderworth — COMMENTED** [review]
Looks great so far! I left some comments with style changes. Ping me for another review once those are addressed.

Also, did you want me to review the entire files or just the changes? I'm happy to help edit the entire files if we want to do that, since this is moving out of "beta" and it may merit such a review... I just don't want to hold things up if you want to merge this quickly.

File: PREPARATION.md
Comment (ag-adampike) [reply PREPARATION.md:27]: I'm in agreement with the superfluous hyphens, but "Container as a Service platforms" reads much better and is more appropriate to me than "Containers as a Service". Azure Container Apps or DigitalOcean App Platform are each platforms that allow customers to host Containers as a Service.

File: PREPARATION.md
Comment (ag-adampike) [reply PREPARATION.md:36]: Same as similar comment above (prefer we keep the word platforms here).

File: do-app-platform-op-cli/README.md
Comment (ag-adampike) [reply do-app-platform-op-cli/README.md:5]: I prefer the second option here (just say App Platform).

File: PREPARATION.md
Comment (Penderworth) [reply PREPARATION.md:27]: I'm happy to use that instead :)

Comment (black-bryan) [reply PREPARATION.md:27]: Combining both of your suggestions here. 

```suggestion
>Containers as a service platforms, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to create a DNS record for these deployments.
```

File: PREPARATION.md
Comment (black-bryan) [reply PREPARATION.md:36]: Combining above suggestions

```suggestion
>Containers as a service platforms, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to use the optional certificate manager component.
```

File: do-app-platform-op-cli/google-workspace/workspace-settings.json
Comment (black-bryan) [reply do-app-platform-op-cli/google-workspace/workspace-settings.json:1]: 🤔 In all the other platform examples, we have defined this file as workspace-settings.json, and tried to clearly instruct admins to edit this file (without the need to rename it) before adding it as a secret.

File: README.md
Comment (Penderworth) [reply README.md:7]: I like this phrasing! Slight adjustments to remove "the" and add a comma:

```suggestion
If you'd prefer to deploy 1Password SCIM Bridge using a marketplace, you can use Google Cloud Platform or DigitalOcean:
```

**Comment by Penderworth** [comment]
Looking through the new articles here at a glance, my suggestions to make them consistent with other READMEs will be fairly vast – not necessarily big changes in wording, but in organization. Mainly, I'd like them to follow the same structure as https://github.com/1Password/scim-examples/blob/main/kubernetes/README.md, which is to include a "Before you begin" section, stepped headers, appendixes, etc.

With that in mind, I think you should go ahead and merge this, and I can create another MR with optimizations to bring these files in line with the other READMEs. That way we don't have to do another review of this with my work, and the new PR will be assigned to me instead of you, @black-bryan. How's that sound?

File: README.md
Comment (Penderworth) [inline README.md:7]: I just spoke with @ag-adampike on a call and we came up with a slightly modified version of this phrasing, since the context and audience are different from AKB. @black-bryan how does this sound?

```suggestion
For an automated deployment of 1Password SCIM Bridge where some settings are pre-configured for you, use a marketplace app from Google Cloud Platform or DigitalOcean:
```

**Review by scottisloud — APPROVED** [review]
Once the `.example` question posed by @grellyd is settled (which I have no opinion on other than that we be consistent with whatever we choose), LTGM!

File: do-app-platform-op-cli/google-workspace/workspace-settings.json
Comment (grellyd) [reply do-app-platform-op-cli/google-workspace/workspace-settings.json:1]: Fair enough. Consistency!

**Review by grellyd — DISMISSED** [review]

**Review by ag-adampike — DISMISSED** [review]

**Review by ag-adampike — APPROVED** [review]
LGTM!

**Review by ag-adampike — APPROVED** [review]

**Review by scottisloud — APPROVED** [review]
LTGM

**Review by scottisloud — APPROVED** [review]

**Review by scottisloud — DISMISSED** [review]

**Review by scottisloud — DISMISSED** [review]

**Review by ag-adampike — APPROVED** [review]

**Review by scottisloud — DISMISSED** [review]

**Review by accraw — DISMISSED** [review]

**Review by accraw — APPROVED** [review]

**Pull request closed without merging**
